src/screens/WebView/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { WebView } from 'react-native-webview';
+
+const RepoView = ({
+  navigation: {
+    state: {
+      params: { reposLink }
```

Comment:
@Chisom, at some points in time complex destructuring, may ship with some weird errors. A typical example would be when `params` is null: it would throw `Can't get reposLink of null or undefined` since `params` should be an object. We have many ways to prevents that from happening, but mostly I define a default value to a parameter that is suspected to throw such errors:

> navigation: {
    state: {
      params: { reposLink } = `{}`
    }
}

Reply:
okay... noted